test/c3/appframework/appframework_test.go

```diff
@@ -68,22 +69,28 @@ var _ = Describe("c3appfw test", func() {
 			   * Create app sources for Cluster Manager and Deployer
 			   * Prepare and deploy C3 CRD with app framework and wait for the pods to be ready
 			   ######### INITIAL VERIFICATIONS #############
-			   * Verify V1 apps are downloaded on Cluster Manager, Deployer and Monitoring Console
```

Comment:
Shouldn't we still have a comment here considering we do still verify apps download state between lines 170-176? Something like "Verify apps went through download state" maybe?

Reply:
Will add the comment